config/options.go

```diff
@@ -1373,21 +1394,18 @@ func (o *Options) Checksum() uint64 {
 
 func (o *Options) applyExternalCerts(ctx context.Context, certsIndex *cryptutil.CertificatesIndex, certs []*config.Settings_Certificate) {
 	for _, c := range certs {
-		cfp := certificateFilePair{}
-		cfp.CertFile = base64.StdEncoding.EncodeToString(c.CertBytes)
-		cfp.KeyFile = base64.StdEncoding.EncodeToString(c.KeyBytes)
-
-		cert, err := cryptutil.ParsePEMCertificateFromBase64(cfp.CertFile)
+		cert, err := cryptutil.ParsePEMCertificate(c.GetCertBytes())
```

Comment:
So to clarify, we would like to rollback the requirements from https://github.com/pomerium/internal/issues/449 ?

We no longer want to enforce the call to `certsIndex.OverlapsWithExistingCertificate(cert)` to prevent console / ingress controller certificates from overriding core certificates?

Reply:
you are right. let's keep it then.

I can add skipping this check in case validation is disabled (certsIndex == nil) in a follow up pr

https://github.com/pomerium/pomerium/blob/205aa8cf530457641b5ef522ef7cd939043d1b71/internal/databroker/config_source.go#L167-L169